docs/cita/configuration_guide/chain-config.md

```diff
@@ -236,3 +239,7 @@ $ rpc getMetaData
 
 ```
 `chainName` 已更新。
+
+[配额管理]: ../system/quota
```

Comment:
Follow the [alphabetical order](https://en.wikipedia.org/wiki/Alphabetical_order)